src/nms.py

```diff
@@ -52,10 +51,11 @@ class CreateUPFParams:
 class NMS:
     """Handle NMS API calls."""
 
-    def __init__(self, url: str):
+    def __init__(self, url: str, ca_certificate_path: str | bool = False):
```

Comment:
We do not seem to be using the `bool` type anywhere, and an empty string is already falsy, so that would be enough.

```suggestion
    def __init__(self, url: str, ca_certificate_path: str = ""):
```